locale/fa/site.json

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
Could you pease check if the translations here for `site.json` against those in #1871?

Reply:
Yes there are against https://github.com/nodejs/nodejs.org/pull/1871

reviews considered in the last commit and more parts translated.

Reply:
So we can merge this PR **instead** of #1871?

Reply:
Yes. :smiley: